### ISSUE TITLE
[Problem/Bug]: Black Screen on video with DRM using keysystem Hardware or recommendation.3000

### ISSUE THREAD

**Issue by slherm** [MicrosoftEdge/WebView2Feedback#4935]
### What happened?

Hi Team,

we are using WebView2 on our streaming app for UWP device (myCANAL french app on windows & xbox).

We use an open source eme/mse player (rx-player) and recently some of our customer have a black screen issue on all videos (but sound is OK).
It's only a part of windows 11 OS (everything is ok on windows 10) and only if we try to use keysystems playready.hardware or playready.recommendation with SL3000
If we switch to playready.software everything is fine but we prefere to use SL3000 keystems to deliver the best quality to our customers.
We have identified a  configutation with this issue : 
- Windows 11 Version 24H2 pour les systèmes x64 (KB5046617)
![image](https://github.com/user-attachments/assets/da3ec34a-3ef8-443c-ab3c-60764949a167)
![image](https://github.com/user-attachments/assets/17335f19-8e06-43a4-ac80-6dc94df3c02b)

Thank a lot for your help & support

### Importance

Blocking. My app's basic functions are not working due to this issue.

### Runtime Channel

Stable release (WebView2 Runtime)

### Runtime Version

130.0.2849.80

### SDK Version

1.0.2365.46

### Framework

WinUI2/UWP

### Operating System

Windows 11

### OS Version

Windows 11 Version 24H2 pour les systèmes x64 (KB5046617)

### Repro steps

Difficult to say :
- on some configurations of window 11 computers 
- video with DRM
- using EME/MSE player in a webview2
- using hardware decoder keysystems
Black screen on video with sound OK

### Repros in Edge Browser

No, issue does not reproduce in the corresponding Edge version

### Regression

Regression in newer Runtime

**Comment by ambikakunnath** [comment]
Thanks for reporting the issue. Can you kindly let me know if this scenario was working earlier? If so, can you kindly share till which version of runtime this was working.

**Comment by vbryh-msft** [comment]
+ @slherm could you please confirm that the same playready settings working correctly in Edge on the same device?

**Comment by slherm** [comment]
> additional question: @slherm could you please confirm that the same playready settings working correctly in Edge on the same device?

the same playready settings works well on Edge on the same device

**Comment by ambikakunnath** [comment]
Thanks for the details. Can you kindly collect and share the traces as per [the instructions here.](https://github.com/microsoft/media-foundation/blob/master/tracing/tracing.md)

**Comment by slherm** [comment]
> Thanks for the details. Can you kindly collect and share the traces as per [the instructions here.](https://github.com/microsoft/media-foundation/blob/master/tracing/tracing.md) Please collect a Simple and Full Trace.

find attached traces
[mftracelog 3.zip](https://github.com/user-attachments/files/17879230/mftracelog.3.zip)

**Comment by ambikakunnath** [comment]
Thanks for providing the logs @slherm . May I also request you to validate if the issue repros with [Win32 Sample app](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2APISample) and [UWP Sample App](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/webview2_sample_uwp) when you navigate to the site in the sample app?

**Comment by champnic** [comment]
Just confirming that the trace is taken of the repro case, and is still being run on a laptop with latest runtime, etc.

**Comment by slherm** [comment]
> Just confirming that the trace is taken of the repro case, and is still being run on a laptop with latest runtime, etc.

yes, I confirm. Trace is taken of the repro case on a laptop with latest runtime

**Comment by slherm** [comment]
Hi team, do you have any update about this issue ?
thank you

**Comment by slherm** [comment]
Hi, our development team have find a solution for this issue. It was on our side. thank you for help and suport.
